contracts/test/utils/parser.js

```diff
@@ -0,0 +1,116 @@
+/* global assert */
+/****** TX RECEIPT PARSERS ******/
+
+/** Returns formatted transaction receipt object with event and arg info
+  * @param {object} txReceipt - transaction receipt object
+  * @returns {object} w/event + args array from txReceipt
+  */
+export const parseTx = (txReceipt) => {
```

Comment:
this spacing is off. everything else uses 2 spaces, this looks like significantly more, maybe your tab settings were wonky for this file?

Reply:
whoops... fixed! 👍